### DISCUSSION TITLE
PLEASE HELP ME

### DISCUSSION THREAD

**Discussion by leons949** [community/community#54960] [Programming Help]
### Select Topic Area

Question

### Body

kali is up to date !! and i cant open the root terminal emulator from the System tools as well but i can open as a root from the normal terminal !!

┌──(root㉿none)-[~]
└─$ sudo systemctl status
● none
State: running
Units: 284 loaded (incl. loaded aliases)
Jobs: 0 queued
Failed: 0 units
Since: Sat 2023-05-06 20:00:21 CEST; 18min ago
systemd: 252.5-2
CGroup: /
├─init.scope
│ └─1 /sbin/init single
├─system.slice
│ ├─ModemManager.service
│ │ └─926 /usr/sbin/ModemManager
│ ├─NetworkManager.service
│ │ └─886 /usr/sbin/NetworkManager --no-daemon
│ ├─accounts-daemon.service
│ │ └─887 /usr/libexec/accounts-daemon
│ ├─colord.service
│ │ └─1383 /usr/libexec/colord
│ ├─containerd.service
│ │ └─977 /usr/bin/containerd
│ ├─cron.service
│ │ └─888 /usr/sbin/cron -f

**Comment by mdxabu** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> kali is up to date !! and i cant open the root terminal emulator from the System tools as well but i can open as a root from the normal terminal !!
> 
> ┌──(root㉿none)-[~] └─$ sudo systemctl status ● none State: running Units: 284 loaded (incl. loaded aliases) Jobs: 0 queued Failed: 0 units Since: Sat 2023-05-06 20:00:21 CEST; 18min ago systemd: 252.5-2 CGroup: / ├─init.scope │ └─1 /sbin/init single ├─system.slice │ ├─ModemManager.service │ │ └─926 /usr/sbin/ModemManager │ ├─NetworkManager.service │ │ └─886 /usr/sbin/NetworkManager --no-daemon │ ├─accounts-daemon.service │ │ └─887 /usr/libexec/accounts-daemon │ ├─colord.service │ │ └─1383 /usr/libexec/colord │ ├─containerd.service │ │ └─977 /usr/bin/containerd │ ├─cron.service │ │ └─888 /usr/sbin/cron -f

use ```sudo -i```
